.gitignore

```diff
@@ -12,3 +12,4 @@ build/
 .watchmanconfig
 coverage
 !packages/cli-platform-ios/src/config/__fixtures__/native_modules/node_modules
+*.env
```

Comment:
Nit: If you add a newline here, you won't get this "invalid" mark:
![image](https://user-images.githubusercontent.com/4123478/223651824-6340ca4b-938a-47da-84c0-eb54ecfec454.png)